onnxruntime/test/python/onnxruntime_test_python.py

```diff
@@ -226,11 +227,12 @@ def runAdvancedTest():
                 runBaseTest2()
                 # raise OSError("could not load any of: " + ' '.join(libnames))
 
-    def testInvalidSetProviders(self):
-        with self.assertRaises(ValueError) as context:
-            sess = onnxrt.InferenceSession(get_name("mul_1.onnx"))
-            sess.set_providers(['InvalidProvider'])
-        self.assertTrue('\'InvalidProvider\' is unavailable' in str(context.exception))
+    #disable this test since we change it from exception to warning
```

Comment:
Then how do we test this scenario?

Reply:
is there anyway we can capture the warning message in current python process?

Reply:
https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertWarns